File: 문제풀이/chap3-4/prob3-4-3.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		// 입력
		Scanner sc = new Scanner(System.in);
		int N = sc.nextInt();
		int[] A = new int[N + 1];
		int[] B = new int[N + 1];
		for (int i = 1; i <= N; i++) A[i] = sc.nextInt();
		for (int i = 1; i <= N; i++) B[i] = sc.nextInt();
		
		// 답 계산/출력
		double Answer = 0.0;
		for (int i = 1; i <= N; i++) {
			Answer += (1.0 / 3.0) * A[i] + (2.0 / 3.0) * B[i];
		}
		System.out.format("%.12f\n", Answer);
	}
}

File: 문제풀이/chap3-4/prob3-4-3.py
# 입력
N = int(input())
A = list(map(int, input().split()))
B = list(map(int, input().split()))

# 답 구하기
Answer = 0.0
for i in range(N):
	Answer += (1.0 / 3.0) * A[i] + (2.0 / 3.0) * B[i]

# 출력
print("{:.12f}".format(Answer))